security/form_login.rst

```diff
@@ -221,11 +221,9 @@ this by setting ``use_referer`` to true (it defaults to false):
 Redirecting on Login Failure
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In addition to redirecting the user after a successful login, you can also set
-the URL that the user should be redirected to after a failed login (e.g. an
-invalid username or password was submitted). By default, the user is redirected
-back to the login form itself. You can set this to a different route (e.g.
-``login_failure``) with the following config:
+After a failed login (e.g. an invalid username or password was submitted), the
+user is redirected back to the login form itself. Use the ``failure_path``
```

Comment:
The parameter with the leading `_` is when using the hidden form field, right? Otherwise you don't have to prefix the `_`.